samples/word/02-paragraphs/insert-in-different-locations.yaml

```diff
@@ -4,95 +4,99 @@ description: Shows how to insert content using the different available locations
+        $("#start").click(() => tryCatch(start));
+        $("#end").click(() => tryCatch(end));
+        $("#after").click(() => tryCatch(after));
+        $("#replace").click(() => tryCatch(replace));
+
+        async function before() {
+            await Word.run(async (context) => {
                 // lets insert before the first paragraph. 
-                var range = context.document.body.paragraphs.getFirst().insertParagraph("This is Before", "before");
+                let range = context.document.body.paragraphs
+                    .getFirst().insertParagraph("This is Before", "before");
                 range.font.highlightColor = "yellow";
-                return context.sync()
-                    .then(function () {
-                        console.log("success");
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+
+                await context.sync();
+                console.log("success");
+            });
         }
 
-        function start() {
-            Word.run(function (context) {
+        async function start() {
+            await Word.run(async (context) => {
                 // This button assumes the before was clicked, so we get the next paragraph and insert text at the begining.Note that there are no-valid locations depending on  the object. For instance insertParagraph and "before" on a paragrpah object is not a valid combination.
-                var range = context.document.body.paragraphs.getFirst().getNext().insertText("This is Start", "start");
+                let range = context.document.body.paragraphs
+                    .getFirst().getNext().insertText("This is Start", "start");
                 range.font.highlightColor = "blue";
                 range.font.color = "white";
-                return context.sync()
-                    .then(function () {
-                        console.log("success");
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+
+                await context.sync();
+                console.log("success");
+            });
         }
 
-        function end() {
-            Word.run(function (context) {
+        async function end() {
+            await Word.run(async (context) => {
                 // Here we go to insert on paragraph
-                var range = context.document.body.paragraphs.getFirst().getNext().insertText(" This is End", "end");
+                let range = context.document.body.paragraphs
+                    .getFirst().getNext().insertText(" This is End", "end");
                 range.font.highlightColor = "green";
                 range.font.color = "white";
-                return context.sync()
-                    .then(function () {
-                        console.log("success");
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+
+                await context.sync();
+                console.log("success");
```

Comment:
This is another case where I wonder if the console.log("success"); is needed. Won't the user see the white-on-green text appear in the document to effectively signal success?

Reply:
local fixed